Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -216,12 +216,11 @@ export default class FilterComponent extends Vue {
 <template>
     <div class="filters-wrapper">
         <div class="filters-width d-none d-md-block">
-            <b-button
+            <hg-button
                 id="filterBtn"
                 class="w-100"
-                :class="{ 'filter-selected': hasFilterSelected }"
                 data-testid="filterDropdown"
-                variant="outline-primary"
+                :variant="hasFilterSelected ? 'highligthed' : 'secondary'"
```

Comment:
`highligthed` should be `highlighted`.

Reply:
Also, I don't actually see a `highlighted` variant defined in hgButton.vue.

Reply:
good catch!